src/models/userModel.js

```diff
@@ -30,4 +31,19 @@ const UserSchema = new mongoose.Schema({
   }]
 });
 
+UserSchema.methods = {
```

Comment:
please do not use this approach, leave business logic to Service rather than on the mongoose model.

Reply:
Will try to move.